README.md

```diff
@@ -6,7 +6,7 @@
 </p>
 
 <p align="center">
-  A privacy-friendly Twitter frontend for mobile devices.
+  A libre privacy-friendly Twitter frontend for mobile devices.
```

Comment:
Could you change this, and the content changes in the "Features" section back to the original text? These are out of scope to this PR 🙂